easybuild/easyconfigs/p/PyTorch/PyTorch-1.10.0_fix-test-dataloader-fixed-affinity.patch

```diff
@@ -0,0 +1,37 @@
+diff -Nru pytorch.orig/test/test_dataloader.py pytorch/test/test_dataloader.py
```

Comment:
Please add decription and author information to this patch.

Reply:
Done